tests/cbor_decode/test5_strange/src/main.c

```diff
@@ -581,6 +581,169 @@ void test_range(void)
 }
 
 
+void test_value_range(void)
+{
+	const uint8_t payload_value_range1[] = {0x86,
+		11,
+		0x19, 0x03, 0xe7, // 999
+		0x29, // -10
+		1,
+		0x18, 42, // 42
+		0x65, 0x77, 0x6f, 0x72, 0x6c, 0x64, // "world"
```

Comment:
Would it be better to write out the characters 

`'e', 'w',` etc

Reply:
Also, why is it prefixed with `'e'`?

Reply:
`0x65` means a tstr with length 5 :)

Reply:
Good idea @hakonfam, done